mod_reforged/hooks/config/character_names.nut

```diff
@@ -0,0 +1,108 @@
+::Const.Strings.CharacterNames.extend([
+    "Agnar",
+    "Alard",
+    "Alaric",
+    "Albert",
+    "Albterus",
+    "Aldwin",
+    "Ansgar",
+    "Badulf",
+    "Baldarich",
+    "Baldo",
+    "Baldovin",
+    "Baldwin",
+    "Baranor",
+    "Bertilo",
+    "Bjorn",
+    "Brando",
+    "Burchard",
+    "Carolus",
+    "Cenric",
+    "Conrad",
+    "Dagfinn",
+    "Dunstan",
+    "Eckhart",
+    "Edgar",
+    "Egino",
+    "Elof",
+    "Emelrich",
+    "Eswig",
+    "Faramund",
+    "Farvald",
+    "Filibert",
+    "Franco",
+    "Friduhelm",
+    "Gaufrid",
+    "Gautvin",
+    "Gerbold",
+    "Gilbert",
+    "Giselbert",
+    "Gismund",
+    "Godric",
+    "Grimm",
+    "Hadubert",
+    "Hagano",
+    "Harold",
+    "Hartwin",
+    "Heidrich",
+    "Helmfried",
+    "Helmo",
+    "Helmut",
+    "Herman",
+    "Kuno",
+    "Lambert",
+    "Landulf",
+    "Lanzo",
+    "Leofric",
+    "Leonard",
+    "Leudagar",
+    "Leuthar",
+    "Leutwin",
+    "Manno",
+    "Meino",
+    "Meinrad",
+    "Milo",
+    "Norbert",
+    "Odo",
+    "Olvir",
+    "Oswin",
+    "Pipin",
+    "Randulf",
+    "Regin",
+    "Reinald",
+    "Reinhard",
+    "Richard",
+    "Roland",
+    "Sigimar",
+    "Silvan",
+    "Siward",
+    "Svenn",
+    "Theo",
+    "Theodar",
+    "Thiemo",
+    "Valdemar",
+    "Veremund",
+    "Waldo",
+    "Waldomar",
+    "Walther",
+    "Widald",
+    "Wido",
+    "Wigand",
+    "Wighart",
+    "Wilfred",
+    "Wolfhart",
+    "Wulfric"
+]);
+
+// This removes all duplicate CharacterNames. This line is redundant for Vanilla + Reforged because our additional names are already checke for duplicates
+// Any mod that adds names after ours may re-introduce duplicates. This is not game-breaking but still undesired
+::Const.Strings.CharacterNames = ::MSU.Array.removeDuplicateStrings(::Const.Strings.CharacterNames);
+
+local knightPrefix = "Sir ";
+foreach(characterName in ::Const.Strings.CharacterNames)
+{
+    ::Const.Strings.KnightNames.push(knightPrefix + characterName);
```

Comment:
Why do you use a local variable `knightPrefix` instead of just `"Sir " + characterName`?

Reply:
Just personal preference at the point in time when I wrote that.